### PR TITLE
Add `caffeine-coroutines` in README

### PR DESCRIPTION
**Pull request by be-hase**
Hello. I always use Caffeine.

I have implemented a library called [caffeine-coroutines](https://github.com/be-hase/caffeine-coroutines), which adds Kotlin coroutines support to Caffeine. If possible, could you add this to the README?

There is also a Kotlin wrapper called [Aedile](https://github.com/sksamuel/aedile), but for the following reasons, I prefer to provide extensions rather than a wrapper.

https://github.com/be-hase/caffeine-coroutines?tab=readme-ov-file#we-will-primarily-focus-on-coroutine-support

### CODE REVIEW

**Comment by ben-manes** [comment]
Sure, but can you make the item length shorter and align into a triangular flow? It’s a bit wordy in comparison

**Comment by ben-manes** [comment]
thanks, merged in manually

**Comment by be-hase** [comment]
thanks. :)
